.github/workflows/build.yaml

```diff
@@ -249,6 +249,12 @@ jobs:
     needs:
       - "rustfmt"
       - "markdown-lint"
+    strategy:
+      matrix:
+        target:
+          - thumbv7m-none-eabi
+          - aarch64-linux-android
+          - aarch64-apple-ios
```

Comment:
Initially I had a separate job for android and ios, but then I realized only the type crates should be built for them since the other ones will transitivly link to the SGX libraries.  I figured it was no harm to also build these as no alloc and not much if any benefit to build them with alloc support.